src/metamorpho/MetaMorphoSnippets.sol

```diff
@@ -182,4 +186,34 @@ contract MetaMorphoSnippets {
             ERC20(IMetaMorpho(vault).asset()).approve(vault, type(uint256).max);
         }
     }
+
+    function reallocateAvailableLiquidity(
```

Comment:
anything we want to add in the description of the function? 
otherwise good to me IMO